contracts/src/v0.8/llo-feeds/v0.4.0/DestinationFeeManager.sol

```diff
@@ -225,9 +225,9 @@ contract DestinationFeeManager is IDestinationFeeManager, ConfirmedOwner, TypeAn
     uint256 numberOfNativeFees;
 
     uint256 feesAndRewardsIndex;
-    bytes32 emptyBytes = bytes32(0);
+    //bytes32 emptyBytes = bytes32(0);
```

Comment:
@Fletch153  I had to comment this out, otherwise the contract does not compile

Reply:
```
Error:
Compiler run failed:
Error: Compiler error (/solidity/libsolidity/codegen/LValue.cpp:56):Stack too deep. Try compiling with `--via-ir` (cli) or the equivalent `viaIR: true` (standard JSON) while enabling the optimizer. Otherwise, try removing local variables.
CompilerError: Stack too deep. Try compiling with `--via-ir` (cli) or the equivalent `viaIR: true` (standard JSON) while enabling the optimizer. Otherwise, try removing local variables.
   --> src/v0.8/llo-feeds/v0.4.0/DestinationFeeManager.sol:240:11:
    |
240 |           poolIds[i],
    |           ^^^^^^^
```

Reply:
ah, ok in that case you can remove it. We could reshuffle the code around but it's not worth it. Stack too deep ultimately means we have too many variables defined within this function